test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +61,12 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
```

Comment:
No need to abstract the assert as separate method. Existing tests do that but that is to avoid repeating `Arrays.asList` everywhere.